docs/installation/060_INSTALL_PATCHMATCH.md

```diff
@@ -18,13 +18,27 @@ Windows systems with no extra intervention.
 
 ## Macintosh
 
-You need to have opencv installed so that pypatchmatch can be built:
+You need to have opencv installed so that pypatchmatch can be built.
 
-```bash
+### Homebrew
+
+```sh
 brew install opencv
 ```
 
-The next time you start `invoke`, after sucesfully installing opencv, pypatchmatch will be built.
+The next time you start `invoke`, after successfully installing opencv, pypatchmatch will be built.
+
+### MacPorts
+
+MacPorts requires some extra steps to get InvokeAI to see the opencv installed files:
+
+```sh
+sudo port install py310-opencv4
+sudo ln -sf /opt/local/lib/opencv4/pkgconfig/opencv4.pc /opt/local/lib/opencv4/pkgconfig/opencv.pc
```

Comment:
the symlink should really be avoided, since when you come to the point where you want to `sudo port uninstall py310-opencv4`, the created symlinks will be left on the system, and can get some "nasty" side-effects (not sure about the syntax since I use brew, but sure you get it 🙈)

When you want to test it out first, just create a fresh venv and install like

```py
python3 -m venv .venv \
    --prompt InvokeAI \
    --upgrade-deps
source .venv/bin/activate
pip3 install --use-pep517 https://github.com/invoke-ai/InvokeAI/archive/refs/heads/main.zip
invoke --web
# will soon be invokeai --web
```

Or use the new installer as soon as it got merged. But I am like 99+% sure that it will work with the MacPorts package as well (it even works with version 2 and version 4)